plugins/range/storage.go

```diff
@@ -5,86 +5,88 @@
 package rangeplugin
 
 import (
-	"bufio"
+	"database/sql"
 	"errors"
 	"fmt"
-	"io"
 	"net"
-	"os"
-	"strings"
 	"time"
+
+	_ "github.com/mattn/go-sqlite3"
 )
 
+func loadDB(path string) (*sql.DB, error) {
+	db, err := sql.Open("sqlite3", fmt.Sprintf("file:%s", path))
+	if err != nil {
+		return nil, fmt.Errorf("failed to open database (%T): %w", err, err)
+	}
+	if _, err := db.Exec("create table if not exists leases4 (mac string not null, ip string not null, expiry int, primary key (mac, ip))"); err != nil {
```

Comment:
the table `lease4` is not configurable, but it might become if we see a use for it